toody-frontend/src/pages/Home/Home.jsx

```diff
@@ -17,12 +17,18 @@ const Home = ({ name }) => {
     // TODO: Complete this function with an axios.get() call
     // to the appropriate backend endpoint for todos. The todos on
     // the homepage should populate correctly when this is done.
+
```

Comment:
actually this line is missing the `getTodos` function declaration for some reason but syntax checker will catch that for you in the future